src/pjpersist/serializers.py

```diff
@@ -28,11 +28,11 @@ def read(self, state):
         return datetime.datetime.strptime(state['value'], self.fmt).date()
 
     def can_write(self, obj):
-        return isinstance(obj, datetime.date)
```

Comment:
Why? I'd keep that `isinstance` because that works for subclasses, whereas `type` will just match an exact instance of a date. AFAIR something (freezing time? freezegun?) subclasses date and datetime to work.

Reply:
Good point, I will revert that change

Reply:
I think it is a valid point but I changed that because in my test I added both date and datetime serializers. Datetime was serialized as a date and that was my motive for that change. The line I'm talking about is 1499:
`>>> serialize.SERIALIZERS = [DateSerializer(), DateTimeSerializer()]`
I think I could revert that change and use `isinstance` and just change the order of serializers but I'm not 100% sure

Reply:
yeah
```
Subclass relationships:

object
    timedelta
    tzinfo
        timezone
    time
    date
        datetime
```
datetime must be first